### PR TITLE
create next.js front-end

### PR DESCRIPTION
**Pull request by Wolfinbarger**
Initialize Next.js app in the app folder.

Response to issue #3

### CODE REVIEW

File: src/app/README.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
+Run the following Next.js setup command
+
+```
+yarn create next-app .
+```
+
+Take the following prompts.
+
+![image](img\prompt-instructions.png)
+
```
Comment (FreakyNobleGas) [inline src/app/README.md:21]: I think we can remove these lines since it's already been completed.

**Review by FreakyNobleGas — CHANGES_REQUESTED** [review]
This looks great! I tested on my machine and it works for me. Just ask that we remove a couple lines from the ReadMe since it'll be confusing later on.

**Review by FreakyNobleGas — APPROVED** [review]
🚀

**Comment by Wolfinbarger** [comment]
I'mma mergin

**Pull request merged**